sdk/lib/rtl/amd64/unwind.c

```diff
@@ -321,6 +419,10 @@ RtlpTryToUnwindEpilog(
         return FALSE;
     }
 
+    // check for popfq
+
+    // also allow end with jmp imm, jmp [target], iretq
+
```

Comment:
Also GCC thinks it's clever to put a NOP as the last instruction when the last function called has the noreturn attribute.

(See condition on line 435, github won't let me comment there)

Reply:
The good thing is, that noreturn functions never return and therefore never execute the epilog :)

Reply:
I'll have to check again when that was a problem...